src/drive/web/modules/drive/Toolbar/components/DocumentQualification.jsx

```diff
@@ -133,42 +23,123 @@ DocumentCategory.propTypes = {
  * its parent
  *
  */
+const filename_extension = '.jpg'
+const id_filename_input = 'filename_input'
```

Comment:
Nit: We should use camel case for consistency. Kind of surprised the linter doesn't pick it up.